syz-manager/manager.go

```diff
@@ -88,8 +91,11 @@ type Manager struct {
 	// Maps file name to modification time.
 	usedFiles map[string]time.Time
 
-	coverFilterFilename string
-	coverFilter         map[uint32]uint32
+	KernelModules      map[string]backend.KernelModule
```

Comment:
Name it `kernelModules` with first lowercase letter as all other fields.

Reply:
rename to modules